docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/attachment/attachment.stories.js

```diff
@@ -20,36 +20,48 @@ export const Attachment = (knobTab) => {
     {
       url: randomUrl(),
       text: 'DOC type document',
+      ext: 'DOC',
+      size: '42.88 KB',
       last_updated: date,
       icon: 'download-file',
     },
     {
       url: randomUrl(),
       text: 'DOCX type document',
+      ext: 'DOCX',
+      size: '32.48 KB',
       last_updated: date,
       icon: 'download-file',
     },
     {
       url: randomUrl(),
       text: 'PDF type document',
+      ext: 'PDF',
+      size: '42.82 KB',
       last_updated: date,
       icon: 'download-file',
     },
     {
       url: randomUrl(),
       text: 'PPT type document',
+      ext: 'PPT',
+      size: '12.88 KB',
       last_updated: date,
       icon: 'download-file',
     },
     {
       url: randomUrl(),
       text: 'XLSX type document',
+      ext: 'XLSX',
+      size: '34.45 KB',
       last_updated: date,
       icon: 'download-file',
     },
     {
       url: randomUrl(),
       text: 'XLS type document',
+      ext: 'XLS',
```

Comment:
please add 1 case for when there is no ext
please add 1 case for when there is no size
please add 1 case for when there is no size and  ext